chart/values.yaml

```diff
@@ -367,6 +368,10 @@ components:
     certbot:
       enabled: false
       email: "certificate@your-domain.com"
+    # A base64ed Diffie-Hellman parameter
+    # This can be generated with: openssl dhparam 4096 2> /dev/null | base64 -w 0
+    # http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_dhparam
```

Comment:
:100:  :+1:

Reply:
With this IMO we do not need to update the docs (also a nice side effect of the DH/cert separation).